docs/sources/panels/expressions.md

```diff
@@ -0,0 +1,189 @@
+
+### Resample
+
+Resample changes the time stamps in each time series to have a consistent time interval. The main use case is so you can resample time series that do not share the same timestamps so math can be performed between them. This can be done by resample each of the two series, and then in a Math operation referencing the resampled variables.
+
+**Fields:**
+
+- **Input -** The variable of time series data (refID (such as `A`)) to resample
+- **Resample to -** The duration of time to resample to, for example `10s`. Units may be `s` seconds, `m` for minutes, `h` for hours, `d` for days, `w` for weeks, and `y` of years.
+- **Downsample -** The reduction function to use when there are more than one data point per window sample. See the reduction operation for behavior details.
+- **Upsample -** The method to use to fill a window sample that has no data points.
+  - **pad** fills with the last know value
+  - **backfill** with next known value
+  - **fillna** to fill empty sample windows with NaNs
+
+## Using an expression as an AlertingNG condition
+
+// KYLE - See earlier comment about whether we should remove this or not. I am generally in favor of removing content that is not applicable to the release, because otherwise we might confuse our users.
+
+### Basic Threshold Alert
+
+Note: When ready, will require `ngalert` feature toggle. The following isn't available yet, but you can follow the following with a Dashboard's Panel edit.
+
+The most common workflow for a basic alert is:
+
+1. Query a backend data source that returns time series (often multiple series uniquely identified by labels).
+2. Reduce each series with a function like mean.
+3. Compare that the reduced (mean) value of each series to a threshold.
+
+To do this with expressions:
+
+1. Make a data source query with a backend data source like the Azure or Prometheus data source. This will be the variable `A` by default (in the future, plan to add the ability to rename these).
+2. Add an expression, Chose the "Reduce" Operation, and put `A` as the Input field. This by default will have the variable `B`.
+3. Add another expression (this will be variable `C`) and chose the math operation. Enter the formula like `$B > 15`
+
+This will return 0 for false, or 1 for true for each item in the response. So variable `C` becomes the selected condition for alerting.
+
+The following shows an example of the above inside the current Dashboard>Panel Edit UI (This UI will likely change, so this may be out of date). The "eye" icon with expressions does not disable a query, but rather stops expressions from returning the data. This show the combined output of `B` and `C` in the Stat panel (Generally, viewing output of numbers with stat or a table view works best, and for series the graph panel).
+
+![image of above description](./condition_example_a.jpg)
```

Comment:
@kylebrandt - If we keep the content, then we need to update and link the images.